heron/packing/src/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPacking.java

```diff
@@ -313,4 +457,22 @@ private int allocateNewContainer(ArrayList<Container> containers) {
     containers.add(new Container(maxContainerResources));
     return containers.size();
   }
+
+  private class IdPair {
```

Comment:
Can we use Pair instead? https://github.com/twitter/heron/blob/master/heron/common/src/java/com/twitter/heron/common/basics/Pair.java#L24